note/src/Note.ts

```diff
@@ -0,0 +1,24 @@
+import './Note.styles.css';
+
+const colors = {
+  info: '#54AEFF66',
+  warning: '#ECE81A66',
+  error: '#FF818266',
+};
+
+/**
+ * Component used to call out information in documentation pages.
+ */
+export class Note extends HTMLElement {
```

Comment:
didn't we agree to write all new component in Lit?
I think we should do it, especially in cases like this where you need inputs (`variant` and `color` properties), Lit can handle it much better and support both property and attribute APIs